docs/pages/blog/turbo-1-5-0.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Turborepo 1.5
+date: 2022/09/19
+description: Turborepo 1.5 brings the Monorepo Handbook, drops the 'run' command, improves pruning, and much more!
+tag: web development
+ogImage: /images/blog/turbo-1-5-0.png
+---
+
+# Turborepo 1.5
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Monday, September 19th, 2022</div>
+
+<Authors authors={[ 'mattpocock', 'gsoltis', 'nathanhammond', 'tknickman', 'anthonyshewdev', 'jaredpalmer', 'mehulkar' ]} />
+
+Turborepo 1.5 is a **huge leap forward for our documentation and DX**, as well as bringing big improvements to `turbo prune`:
+
+- [**The Monorepo Handbook**](#the-monorepo-handbook): We've built the missing manual for your monorepo - a guide on workspaces, code sharing, integrating common tools and much more.
+- [**Drop the `run`**](#drop-the-run): `turbo run <task>` can now be shortened to `turbo <task>`
+- [**Prune now supported on pnpm, yarn and npm**](#prune-now-supported-on-npm-pnpm-and-yarn): Pruning your monorepo is now supported in all major package managers.
```

Comment:
Wow cool, which versions of NPM will be supported? I haven’t spotted PRs related to it yet 🕵️

Reply:
cc @nathanhammond

Reply:
@weyert Looks like I might have jumped the gun on this one, although I do think it's being actively worked on.

Reply:
All good :) Looking forward when it is ready to use it as a bridge/holdover until we can use PNPM with Gitlab